### ISSUE TITLE
Comments in a non-semicolon style have extra indent

### ISSUE THREAD

**Issue by timothyeburke** [beautify-web/js-beautify#815]
An inline comment on the line immediately following a line without a line terminator, the inline comment gets an extra indent. Legit bug?

Input:

```
foo()
// this is a comment
bar()

const foo = 5
// comment
bar()

```

Output:

```
foo()
    // this is a comment
bar()

const foo = 5
    // comment
bar()

```

**Comment by bitwiseman** [comment]
Support for newline as start of new statement is still a bug farm for the beautifier. Stuff like this.

**Comment by bitwiseman** [comment]
Semicolon-less javascript has been problematic for a long time.  Nailing this case done will have to wait for the next release.

**Comment by bitwiseman** [comment]
This would also benefit from the fix proposed in #1069.  Comments would match the indent of the second element.
